components/compliance-service/ingest/ingestic/ingestic.go

```diff
@@ -500,6 +516,12 @@ func (backend *ESClient) JobStatus(ctx context.Context, jobID string) (project_u
 
 	percentageComplete := getPercentageComplete(tasksGetTaskResponse.Task.Status)
 
+	// If the task is marked complete but the percentage complete is not 1 then the task stopped unexpectedly
+	if tasksGetTaskResponse.Completed && percentageComplete != 1 {
+		return project_update_lib.JobStatus{}, fmt.Errorf("Task %s stopped unexpectedly. "+
+			"For more information go to http://localhost:10141/.tasks/task/%s", jobID, jobID)
```

Comment:
This seems like it might not be the best info in some multi-node world but it's fine for now.